tracker/iou_tracker.py

```diff
@@ -280,13 +287,24 @@ def _associate_instances_to_tracks(
     return list_of_matches, unmatched_instances, unmatched_tracks
 
 
-def read_panoptic_image(path: Text, label_divisor: int) -> np.ndarray:
+def read_panoptic_image_2ch(path: Text, label_divisor: int) -> np.ndarray:
   """Reads in a panoptic image in 2 channel format and returns as np array."""
   with tf.io.gfile.GFile(path, 'rb') as f:
     image = tf.cast(tf.io.decode_image(f.read()), tf.int32).numpy()
+
+  if image.shape[2] == 3 and np.any(image[..., 2] != 0):
```

Comment:
if image.shape[2] != 2?
more explicit?

Reply:
We still want to support standard RGB images that have only zeros in the third channel. This enables much higher support, as not all tools/libraries support other image formats. Please note, vis.store_raw_predictions will also always store an RGB image even though, the third channel is empty when using 'two_channel_png'.

Reply:
SG! Thanks for the clarification, Mark!